nesta/core/schemas/tier_1/tests/test_ontology.py

```diff
@@ -21,7 +21,7 @@ def test_ontology_uniqueness(ontology):
 def test_validate(ontology):
     dirname = Path(os.path.dirname(__file__)).parent
     dataset_dirname = os.path.join(dirname, 'datasets')
-    firsts, middles, lasts = [], [], []
+    firsts, middles, lasts = set(), set(), set()
```

Comment:
sets instead of lists, since the assert error message is easier to read when duplicates aren't in